.gitignore

```diff
@@ -4,4 +4,5 @@
 /target/
 /core/target/
 /modules/enumeratum/target
+/modules/refined/target
```

Comment:
what about:

```suggestion
target/
```

Reply:
only if we never use the medeia.target folder XD but we won't ;) good suggestion